pkg/abi/abi.go

```diff
@@ -315,6 +315,56 @@ func (a ABI) Errors() map[string]*Entry {
 	return m
 }
 
+// Returns the components value from the parsed error
+func (a ABI) ParseError(revertData []byte) (*Entry, *ComponentValue, bool) {
+	return a.ParseErrorCtx(context.Background(), revertData)
```

Comment:
Pass in context instead of `context.Background?`

Reply:
It seems the convention in this library has become to provide two flavors of each method - one that propagates a context and one that uses `context.Background()`. I'm not totally sure of the history, but I see the same pattern followed here.